src/proto/grpc/testing/messages.proto

```diff
@@ -210,9 +207,28 @@ message LoadBalancerStatsRequest {
   int32 num_rpcs = 1;
   // If num_rpcs have not completed within timeout_sec, return partial results.
   int32 timeout_sec = 2;
+  // response header+trailer we want the values of
+  repeated string metadata_keys = 3;
 }
 
 message LoadBalancerStatsResponse {
+  message MetadataEntry {
+    string key = 1;
+    string value = 2;
+  }
+  message RpcMetadata {
+    // Metadata associated with the RPC call
+    repeated MetadataEntry metadata = 1;
+  }
+  message MetadataByPeer {
+    // metadata values in the response for each rpc for all keys
+    // specified in LoadBalancerStatsRequest.metadata_keys
+    // Binary values are base64 encoded
+    // note this is a list of maps: each element in the list
```

Comment:
The comment doesn't match the definition. There is no map as far as I can see. This is really a list of lists. The outer list is list of RPCs and the inner list is the list of key+value pair. Is that right? If correct can you fix the comment so it correctly describes the proto?

Reply:
Modified. PTAL.